src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -80,13 +80,16 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public HttpClientHandler() { }
         public bool AllowAutoRedirect { get { return default(bool); } set { } }
         public System.Net.DecompressionMethods AutomaticDecompression { get { return default(System.Net.DecompressionMethods); } set { } }
+        public bool CheckCertificateRevocationList { get { return default(bool); } set { } }
         public System.Net.Http.ClientCertificateOption ClientCertificateOptions { get { return default(System.Net.Http.ClientCertificateOption); } set { } }
+        public System.Security.Cryptography.X509Certificates.X509CertificateCollection ClientCertificates { get { return default(System.Security.Cryptography.X509Certificates.X509Certificate2Collection); } }
```

Comment:
What's the rationale for having the API defined with a return type of ```X509CertificateCollection``` rather than ```X509Certificate2Collection```?  Does doing so afford us more flexibility in some regard?